docs/src/components/ComponentDoc/ComponentDoc.tsx

```diff
@@ -91,10 +91,10 @@ class ComponentDoc extends React.Component<ComponentDocProps, ComponentDocState>
 
   /* TODO: bring back the right floating menu
```

Comment:
Can we just remove these comments.

Reply:
Sure, separate PR since it is not touched in this one.